cannon/mipsevm/tests/evm_common_test.go

```diff
@@ -1010,25 +854,16 @@ func TestEntryEVM(t *testing.T) {
 	}
 }
 
-func TestEVMSingleStepBranch(t *testing.T) {
-	versions := GetMipsVersionTestCases(t)
-	cases := []struct {
-		name         string
-		pc           Word
-		expectNextPC Word
-		opcode       uint32
-		regimm       uint32
-		expectLink   bool
-		rs           arch.SignedInteger
-		rt           Word
-		offset       uint16
-	}{
+func TestEVM_SingleStep_Branch32(t *testing.T) {
+	testutil.Cannon32OnlyTest(t, "These tests are fully covered for 64-bits in TestEVM_SingleStep_Branch64")
+	t.Parallel()
+	cases := []branchTestCase{
 		// blez
 		{name: "blez", pc: 0, opcode: 0x6, rs: 0x5, offset: 0x100, expectNextPC: 0x8},
 		{name: "blez large rs", pc: 0x10, opcode: 0x6, rs: 0x7F_FF_FF_FF, offset: 0x100, expectNextPC: 0x18},
 		{name: "blez zero rs", pc: 0x10, opcode: 0x6, rs: 0x0, offset: 0x100, expectNextPC: 0x414},
 		{name: "blez sign rs", pc: 0x10, opcode: 0x6, rs: -1, offset: 0x100, expectNextPC: 0x414},
-		{name: "blez rs only sign bit set", pc: 0x10, opcode: 0x6, rs: testutil.ToSignedInteger(0x80_00_00_00), offset: 0x100, expectNextPC: 0x414},
+		{name: "bexpectRevertlez rs only sign bit set", pc: 0x10, opcode: 0x6, rs: testutil.ToSignedInteger(0x80_00_00_00), offset: 0x100, expectNextPC: 0x414},
```

Comment:
oops - looks like a find/replace typo :D
```suggestion
		{name: "blez rs only sign bit set", pc: 0x10, opcode: 0x6, rs: testutil.ToSignedInteger(0x80_00_00_00), offset: 0x100, expectNextPC: 0x414},
```